Configuration/Realm/Realm.xcconfig

```diff
@@ -27,11 +27,11 @@ LD_RUNPATH_SEARCH_PATHS[sdk=iphone*] = $(inherited) @executable_path/Frameworks
 LD_RUNPATH_SEARCH_PATHS[sdk=watch*] = $(inherited) @executable_path/Frameworks @loader_path/Frameworks;
 LD_RUNPATH_SEARCH_PATHS[sdk=appletv*] = $(inherited) @executable_path/Frameworks @loader_path/Frameworks;
 
-OTHER_LDFLAGS[sdk=macosx*] = -lrealm$(REALM_LIBRARY_SUFFIX);
+OTHER_LDFLAGS[sdk=macosx*] = -lrealm$(REALM_LIBRARY_SUFFIX) -framework SystemConfiguration;
```

Comment:
We don't have separate targets for different platforms in Xcode project and watchOS doesn't have this framework so I don't see a way how to specify it there.

Good point about static framework, let me think a bit more about that.